docs/experiments-car17v1.5.md

```diff
@@ -11,7 +11,7 @@ lucene-index.car17v1.5.pos+docvectors+rawdocs -storePositions -storeDocvectors \
 -storeRawDocs >& log.car17v1.5.pos+docvectors+rawdocs &
 ```
 
```

Comment:
nit: CAR is all upper case.

Reply:
I think for all the file names we downcase? E.g., robust04 not Robust04, nyt instead of NYT, etc.